### DISCUSSION TITLE
Is there an URL parameter to display pages in "embed" mode (without navigation menu nor header)

### DISCUSSION THREAD

**Discussion by Guts** [squidfunk/mkdocs-material#2390] [Q&A] [answered]
Hello,

I think the title is quite clear enough but I would like to know if there is some URL parameters impacting how the page is rendered in a browser.

Something like `https://doc.awesome-tool.app/module-x-explained.html?nav=false&header=false`? I saw it's possible [to hide navigation using page.meta](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#hide-the-sidebars), so maybe there is some hidden way?

Use case: in an application, open the page related to the actual plugin using the in-browser web view (Qt 5), disabling navigation or headers

**Comment by squidfunk** [comment, marked as answer]
Nope, no such mode, as we haven’t had that use case. Should be achievable with some JS + CSS.

**Reply by Guts** [reply]
Thanks for your quick reply.

Should I convert it into an idea as proposal of future feature? It could be useful to get page.meta specific settings replicated as URL parameters no?

> Should be achievable with some JS + CSS.

I'm not good enough in JS to perform this, so if you have some quick leads to follow, I'll give it a try.

**Reply by squidfunk** [reply]
You can create an issue with some use cases, so we can see if it would be useful to other users. If it is, we can implement it!

**Reply by Guts** [reply]
> You can create an issue with some use cases, so we can see if it would be useful to other users. If it is, we can implement it!

Done here: https://github.com/squidfunk/mkdocs-material/issues/2396
